README.md

```diff
@@ -23,19 +23,30 @@ EntandoDeBundles served in the cluster can be installed in an EntandoApp using t
-| DB_VENDOR                                                     | Which database will be used. Default `postgres`                                                               |
-| DB_HOST                                                       | Database host. Default `localhost`                                                                            |
-| DB_PORT                                                       | Database port. Default `5432`                                                                                 |
-| DB_NAME                                                       | Database name. Default `digital_exchange`                                                                     |
-| DB_OPTIONS                                                    | Database options. Default `useSSL=false`                                                                      |
-| DB_USER                                                       | Database user. Default `admin`                                                                                |
-| DB_PASS                                                       | Database password. Default `admin`                                                                            |
-| ENTANDO_URL                                                   | The URL to access the Entando App instance.                                                                   |
-| ENTANDO_APP_NAMESPACE                                         | The kubernetes namespace where the entando app is running. Default to `test-namespace`;                       |
-| ENTANDO_APP_NAME                                              | The entando app name that this service is in. Defaults to `test-entando`.                                     |
-| ENTANDO_BUNDLE_TYPE                                           | The bundle type that should be handled by this service. It can be `git` or `npm`. The default value is `git`. |
+| Group | Name | Value [default] | Description |
+| :---   | :--- | :--- |:--- |
+| General | ENTANDO_URL                 |  | The URL to access the Entando App instance                            |
+|     | ENTANDO_APP_NAMESPACE                                        | [test-namespace] | The kubernetes namespace where the entando app is running |
+|     | ENTANDO_APP_NAME                                             | [test-entando] | The entando app name that this service is in |
+|    | ENTANDO_ECR_DEAPP_REQUEST_RETRIES    | [3] |  Number of times the CM retries the component create/update before giving up     |
+|    | ENTANDO_ECR_DEAPP_REQUEST_BACKOFF       | [5] |  Seconds to wait before the next attempt is executed     |
+|    | ENTANDO_ECR_POSTINIT        |  |  Configuration of the postinit process      |
+|    | ENTANDO_CONTAINER_REGISTRY_CREDENTIALS   | [null] | Credentials for each container registry    |
```

Comment:
Is null really the default?

Reply:
`@Value("${entando.container.registry.credentials:#{null}}")` Yes, it

Reply:
changed to blank